server/data/data-ldap-integration-testing/src/test/java/org/apache/james/user/ldap/ReadOnlyUsersLDAPRepositoryTest.java

```diff
@@ -177,4 +178,10 @@ public void containsWithGetUserShouldBeTrueWithVirtualHosting() throws Exception
         startUsersRepository(ldapRepositoryConfigurationWithVirtualHosting());
         assertThat(ldapRepository.contains(ldapRepository.getUser(new MailAddress(JAMES_USER_MAIL)))).isTrue();
     }
+
+    @Test
+    public void userInputShouldBeSanitized() throws Exception {
```

Comment:
this test doesn't make sense to me. We should try to describe a bit more how it's malicious and what we are protecting ourselves against.

Reply:
I try to make the test more explicit. I am not sure I success